gradle-baseline-java-config/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -21,8 +21,5 @@
     <suppress files="\.(bdr|eot|gif|gzip|jar|json|otf|png|svg|ttf|woff|zip)$" checks="NewlineAtEndOfFile" />
 
     <!-- Generated code should not be subjected to checkstyle. -->
-    <suppress files="[/\\].*[/\\]generated[/\\]" checks="." />
-    <suppress files="[/\\].*[/\\]generated_schema[/\\]" checks="." />
-    <suppress files="[/\\].*[/\\]generated_src[/\\]" checks="." />
-    <suppress files="[/\\].*[/\\]generated_testSrc[/\\]" checks="." />
+    <suppress files="[/\\].*[/\\]generated.*[/\\]" checks="." />
```

Comment:
this would allow people to generate code to `generated_src`, `generated-src` and `generated_src`...

Thoughts on forcing people to pick one and run with it?

Reply:
I've seen `generated_*` and been generally happy with that

Reply:
:+1: as long as we also tolerate `src/generated/java` which I think immutables does :)

Reply:
Do you want stricter enforcement here? Or are we ok saying "any directory that begins with 'generated' will be excluded from checkstyle"?